pkg/streaming/commands/processor_list.go

```diff
@@ -135,10 +133,21 @@ func (opts *ProcessorListOptions) printColumns() []metav1beta1.TableColumnDefini
 		{Name: "Name", Type: "string"},
 		{Name: "Function", Type: "string"},
 		{Name: "Inputs", Type: "string"},
-		{Name: "Input names", Type: "string"},
 		{Name: "Outputs", Type: "string"},
-		{Name: "Output names", Type: "string"},
 		{Name: "Status", Type: "string"},
 		{Name: "Age", Type: "string"},
 	}
 }
+
+func prependParameterNames(streams, parameters []string) []string {
+	result := make([]string, len(streams))
+	for i, streamName := range streams {
+		parameterName := parameters[i]
+		if parameterName == "" {
```

Comment:
Because of defaults, it's better to reverse the if and test for `parameterName != streamName`.
Will fix.